i18n/ko.json

```diff
@@ -268,110 +280,112 @@
     "sortTime": "길이",
     "sortTimesPlayed": "재생 횟수",
     "sortYear": "연도",
-    "tagExclusion": "태그가 없습니다.",
-    "tagIntersection": "태그 포함",
-    "tagUnion": "태그는",
+    "tagExclusion": "태그 제외하기",
+    "tagIntersection": "태그",
+    "tagUnion": "태그 조합",
     "totalFolders": "폴더",
     "totalFoldersPlural": "폴더",
-    "videos": "비디오"
+    "videos": "동영상"
   },
   "STATISTICS": {
     "average": "평균",
     "clipHeight": "클립 크기",
-    "doNotExtract": "추출하지 않음",
-    "files": "파일들",
+    "doNotExtract": "선택하지 않음",
+    "files": "파일",
     "fileSizes": "파일 크기",
-    "folderNotLive": "비디오 폴더가 연결되지 않았습니다.",
+    "folderNotLive": "동영상 폴더가 연결되지 않음",
     "general": "일반",
-    "heading": "현재허브의 통계 :",
+    "heading": "현재 Hub에 대한 통계:",
     "largest": "가장 큰",
     "longest": "가장 긴",
-    "minutes": "분",
-    "numOfClips": "비디오 클립정보",
+    "minutes": "분마다",
+    "numOfClips": "동영상 클립 조각",
     "numOfFiles": "파일 수",
     "numOfFolders": "폴더 수",
-    "numOfThumbs": "썸네일 수",
-    "oneEvery": "하나 하나",
-    "perVideo": "비디오 당",
+    "numOfThumbs": "섬네일 수",
+    "oneEvery": "1개",
+    "perVideo": "개 동영상1개에",
     "screenHeight": "스크린샷 크기",
     "seconds": "초",
     "settings": "추출 설정",
-    "shortest": "최단",
+    "shortest": "가장 짧은",
     "smallest": "가장 작은",
-    "snippetLength": "클립 조각의 길이",
+    "snippetLength": "클립 길이",
     "total": "합계",
-    "videoFolder": "여기에있는 비디오 :",
-    "videoLengths": "비디오 길이"
+    "videoFolder": "동영상 위치:",
+    "videoLengths": "동영상 길이"
   },
   "SYSTEM": {
     "cancel": "취소",
-    "continueAnyway": "어쨌든 계속",
-    "directory": "디렉토리",
-    "doesNotExist": "없습니다.",
-    "hubAlreadyExists": "같은 이름으로 허브가 존재합니다.",
-    "noSuchFileFound": "해당파일을 찾을 수 없습니다. :",
-    "pleaseChangeName": "허브이름을 변경하십시오.",
-    "selectDefaultPlayer": "선호하는 비디오 플레이어를 선택하십시오.",
-    "selectPreviousHub": "이전에 저장된 허브파일을 선택하십시오.",
-    "selectRootFolder": "소스비디오 폴더 선택",
-    "videoFolderNotFound": "소스비디오 폴더를 찾을 수 없습니다. :"
+    "continueAnyway": "무시하고 계속",
+    "directory": "디렉터리",
+    "doesNotExist": "존재하지 않습니다.",
+    "HubAlreadyExists": "같은 이름의 Hub가 있음",
```

Comment:
this got accidentally capitalized `h` -> `H` -- I think it needs to be exact, else it will not work

Reply:
It's my mistake. hub -> Hub

I changed all the hubs on the left. Hub ->hub

Use the Hub on the right?
I think it would be nice to use the name of the Video-Hub-App.